CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # go-ipfs changelog
 
+- Accept named fifos ( pipes ) when supplied directly as an argument, continues to reject them when encountered recursively
```

Comment:
Ah, let's just delete this file. This was the go-ipfs changelog and somehow snuck into this repo when the commands lib package was extracted.